spec/models/base_organisation_spec.rb

```diff
@@ -56,5 +56,37 @@
       org.save
     end
   end
+  
+  describe 'friendly_id' do
+    # TODO: more tests on friendly_id methods?
```

Comment:
Yes, I clearly misunderstood you. I was thinking if e.g. we don't need some
tests which check which methods are called and so on. For now I'm just
testing if the slugs are setup correctly. The question to you is: should be
check something else or that's enough?

2016-05-10 9:58 GMT+02:00 Sam Joseph <notifications@github.com>:

> In spec/models/base_organisation_spec.rb
> <https://github.com/AgileVentures/LocalSupport/pull/292#discussion_r62626919>
> :
>
> > @@ -56,5 +56,37 @@
> >        org.save
> >      end
> >    end
> > +
> > +  describe 'friendly_id' do
> > +    # TODO: more tests on friendly_id methods?
>
> sure, but I think you misunderstood my question.
>
> Your comment says "# TODO: more tests on friendly_id methods?" and I'm
> responding to that saying, "do we need more tests?". You've answered as if
> I asked "do we have too many tests?". The question is "do we *need*
> more?" and if the answer is "no" then we can delete the comment right? :-)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/AgileVentures/LocalSupport/pull/292/files/bf7b03ed1d65dedf261c812acd715258d6344307#r62626919>
>